### DISCUSSION TITLE
is there tools that converts the openfoam's grid to su2

### DISCUSSION THREAD

**Discussion by Zcaic** [su2code/SU2#2041] [Q&A]
hello, now I generate a Cartesian grid used snappyhexmesh，how can I use it in SU2？ Is there any  tools that converts the openfoam's grid to SU2？

**Comment by bigfooted** [comment]
Snappy is really for openfoam. It's better to build su2 meshes in gmsh than try to convert it. Not all meshes can be translated into su2, for instance meshes with hanging nodes. We currently do not have a conversion tool for this. SU2 can read cgns files and it's own native SU2 file format.
